<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BGSLibrary QT GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_algorithms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithms</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_algorithms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_inputdata">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input (specify a video file or a directory path containing the sequence of images)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inputdata">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>721</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>./dataset/video.avi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_inputdata">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>30</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_webcamera">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use web camera</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_webcamera">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_imageseq">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence of images?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_startat">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>60</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start at:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_startat">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_stopat">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>85</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop at:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_stopat">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>85</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>250</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_in">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>IMG_INPUT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_fg">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>IMG_FOREGROUND</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_img_bg">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>IMG_BACKGROUND</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_foreground">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>250</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Foreground mask</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>250</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background model</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_save_im">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_save_fg">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_save_bg">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>470</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>550</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_exectime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estimated FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fps_txt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_framenumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_framenumber_txt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>470</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frameres">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frameresw_txt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_frameresh_txt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_frameres_2">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>440</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_out_in">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>400</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>./output/in/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_out_in">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_out_fg">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>400</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>./output/fg/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_out_fg">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_out_bg">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>400</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_out_bg">
    <property name="geometry">
     <rect>
      <x>868</x>
      <y>400</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>./output/bg/</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_kfn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep frame number</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBGSLibrary">
    <property name="title">
     <string>BGSLibrary</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuBGSLibrary"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
